### PR TITLE
Records Table, and Sidebar Themeing

### PR DESCRIPTION
**Pull request by jiexi**
Styled records table( incl. sort by type menu), and sidebar

@wtachau

### CODE REVIEW

File: Medlink/InboxViewController.m [hunk 1]
```diff
@@ -153,10 +154,20 @@ -(NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
...
+    cell.selectionStyle = UITableViewCellSelectionStyleNone;
     if (indexPath.row == 0) {
-      cell.textLabel.text = [NSString stringWithFormat:@"Records sorted by: %@", [MedLinkHelperMethods textForSortType:self.sortType]];
+      cell.textLabel.text = [NSString stringWithFormat:@"RECORDS SORTED BY: %@", [MedLinkHelperMethods textForSortType:self.sortType]];
+      cell.textLabel.font = [UIFont fontWithName:@"MuseoSansRounded-700" size:11.0];
     } else {
       cell.textLabel.text = [MedLinkHelperMethods textForSortType:(InboxSortType)(indexPath.row - 1)];
+      if(self.sortType != (indexPath.row - 1)){
+        cell.textLabel.textColor = [UIColor darkGrayColor];
+        cell.textLabel.font = [UIFont fontWithName:@"MuseoSansRounded-700" size:13.0];
```
Comment (wtachau) [inline Medlink/InboxViewController.m]: Check out how Sophie de-localized the font name (look at LoginOpeningView line 44)

**Comment by wtachau** [comment]
Nice work! This got a little muddled since I merged in Sophie's code for the sake of the demo. Let's fix the one font name issue I mentioned and we can get this merged in. @jiexi

**Comment by jiexi** [comment]
@wtachau, fixed. Went with the font, font_bold, font_bolder convention.

**Comment by wtachau** [comment]
:+1: lgtm. nice work!

**Pull request merged**